content/javascript/concepts/arrays/arrays.md

```diff
@@ -22,27 +22,25 @@ const numberArray = [0, 1, 2, 3];
 const mixedArray = [1, 'chicken', false];
 ```
 
-## Creating an Array
+## Creating an array
```

Comment:
Hi, I just talked to Sonny about this the other day. In "doc" entries, subheadings are to be in title case, like this:
```suggestion
## Creating an Array
```